velox/exec/Spiller.h

```diff
@@ -218,6 +223,13 @@ class Spiller {
           spilledPartitions(_spilledPartitions) {}
 
     Stats() = default;
+
+    Stats& operator+=(const Stats& other) {
+      spilledBytes += other.spilledBytes;
+      spilledRows += other.spilledRows;
+      spilledPartitions = std::max(spilledPartitions, other.spilledPartitions);
```

Comment:
Why is this a max and not a plus?

Reply:
Hmm, I were thinking when we sum up the spill stats from different operators and we kind of take the max as we spill the same set of partitions. It doesn't seem right to take a max also. I change to plus which is more meaningful. Thanks for pointing out.

Reply:
After the change, should we expect spilledPartitions to be equal to total number of partitions times the number of drivers at the end of the query?

Reply:
Yes